core/src/main/java/com/pixelkobold/world/Worlds.java

```diff
@@ -30,5 +36,64 @@ public static void loadAll() {
 //			worldMap.put(file.nameWithoutExtension(), worldData.toWorld());
         }
 
+
+    }
+
+
+    private static Array<String> getMapsArray(String path) {
+        FileHandle dir = Gdx.files.internal(path);
+        Object[] files = Arrays.stream(dir.readString().split("\n")).filter(file -> file.endsWith("tmx")).toArray();
```

Comment:
No need to use streams if you're going to loop over everything anyway. Use a simple `for-each` loop